Changelog.md

```diff
@@ -1,28 +1,32 @@
 #  Odo Changelog
 
-## Next
+## 2.3.0
 
 ### Feature/Enhancements
 
-- Support for `preStart` lifecycle events ([#4901](https://github.com/openshift/odo/issues/4901))
-
 ### Bug Fixes
 
-- `odo push` now displays "Waiting for component to start" only once ([4919](https://github.com/openshift/odo/pull/4919))
-
 ### Tests
 
 ### Documentation
 
-## 2.2.x
+## 2.2.4
 
 ### Feature/Enhancements
 
+- Support for `preStart` lifecycle events ([#4901](https://github.com/openshift/odo/issues/4901))
+- Do not cancel the whole execution if one of the test suite fails ([#4943](https://github.com/openshift/odo/issues/4943))
```

Comment:
```suggestion
- Support for `preStart` lifecycle events ([#4901](https://github.com/openshift/odo/issues/4901))
- Do not cancel the whole execution if one of the test suite fails ([#4943](https://github.com/openshift/odo/issues/4943))
- Allow linking using ServiceBinding without ServiceBindingOperator ([#4543](https://github.com/openshift/odo/issues/4543)).
```

Reply:
Also, add the following:
```markdown
- Add Sonar Cloud config file: sonar-project.properties([#4900](https://github.com/openshift/odo/pull/4900))
```

Reply:
@mohammedzee1000 Missed out on this one.

Reply:
updated, thanks